operator/v1/types.go

```diff
@@ -107,6 +107,13 @@ type RegistrationConfiguration struct {
 	//  	he can set featuregate/Foo=false before upgrading. Let's say the cluster-admin wants featuregate/Foo=false.
 	// +optional
 	FeatureGates []FeatureGate `json:"featureGates,omitempty"`
+
+	// ClusterAnnotations is a set of annotations that agent used when creating a managed cluster.
+	// After the managed cluster is created, modify this field won't affect the managed cluster any more.
```

Comment:
also need a const definition of this annotationPrefix

Reply:
Yes, adding one now...

Reply:
Added.

Reply:
After the managed cluster is created, modify this field won''t affect the managed cluster any more. But if the managed cluster is deleted from the hub, it will be used again when agent attempt to create a managed cluster. The annotations must has prefix: "agent.open-cluster-management.io", otherwise, it will be ignored.'

I do not think you still need these.

Reply:
Agree, the content deleted.